src/test/java/Sprint2Tests.java

```diff
@@ -0,0 +1,30 @@
+import Pages.LandingPage;
+import Pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Sprint2Tests extends BaseTest {
+    //Story: Create a new playlist
+
+
+
+   @Test
+    public void addSongsToFavorites() {
+        LoginPage loginPage = new LoginPage(driver);
+        LandingPage landingPage = new LandingPage(driver);
+        loginPage.logIn("chelsea.laurenson@testpro.io", "Koelapp1!!");
+        //landingPage.clickHeartFirstRecentlyPlayedSongHomePage();
+        landingPage.clickAllSongs();
+        landingPage.likeFirstSongInAllSongs();
+        landingPage.likeSecondSongInAllSongs();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
Please, add Assert

**One test - One Assert**

For example


![Clean_Test](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/6d341dc5-b9f0-4d56-ac28-3ea676c8ef61)